File: tests/expected/go/global2.go
package main

import (
	"fmt"
	"github.com/adsharma/py2many/pygo/runtime"
)

var Code0 int = 0
var Code1 int = 1
var CodeA string = "a"
var CodeB string = "b"
var LB = map[string]bool{CodeA: true}
var LC = map[string]int{CodeB: Code0}

func main() {
	if !(pygo.MapContains(LB, "a")) {
		panic("assert")
	}
	fmt.Printf("%v\n", "OK")
}

File: tests/expected/julia/global2.jl
code_0 = 0
code_1 = 1
code_a = "a"
code_b = "b"
l_b = Set([code_a])
l_c = Dict(code_b => code_0)
if abspath(PROGRAM_FILE) == @__FILE__
    @assert("a" ∈ l_b)
    println("OK")
end
